Documentation/doc/Documentation/Installation.txt

```diff
@@ -508,6 +508,15 @@ and the \ref PkgRidges_3 packages.
 
 The \sc{Eigen} web site is <A HREF="http://eigen.tuxfamily.org">`http://eigen.tuxfamily.org`</A>.
 
+\subsection thirdpartyLapack Lapack
+
+\sc{Lapack} is a `Fortran` library for linear algebra.
+
+In \cgal, \sc{Lapack} is an alternative to \sc{Eigen} for singular value decomposition for the \ref
+PkgJet_fitting_3 and the \ref PkgRidges_3 packages.
+
+The \sc{Lapack} web site is <A HREF="http://www.netlib.org/lapack/">`http://www.netlib.org/lapack/`</A>.
+
```

Comment:
The interface is still here for backward compatibility but I think even in our cmake scripts we removed all mention to this library. The main reason for not documenting it is that we do not test it so we cannot support it.